packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -148,6 +166,13 @@ class RealAccountSignup extends Component {
 
     closeModalWithHooks = () => {
         this.closeModal();
+        setTimeout(() => {
+            const post_signup = JSON.parse(sessionStorage.getItem('post_real_account_signup'));
```

Comment:
This should be unnecessary, as it's there in the MT5 store as `realAccountSignupEndListener`. This is the one we discussed in the Core PR.

Reply:
Done.